src/main/sensors/gyroanalyse.c

```diff
@@ -117,7 +117,7 @@ void gyroDataAnalyseInit(uint32_t targetLooptimeUs)
     for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
         fftResult[axis].centerFreq = 200; // any init value
         biquadFilterInitLPF(&fftFreqFilter[axis], DYN_NOTCH_SMOOTH_FREQ_HZ, looptime);
-        biquadFilterInit(&fftGyroFilter[axis], FFT_BPF_HZ, 1000000 / FFT_SAMPLING_RATE_HZ, 0.001f * gyroConfig()->dyn_notch_quality, FILTER_BPF);
+        biquadFilterInit(&fftGyroFilter[axis], FFT_BPF_HZ, 1000000 / FFT_SAMPLING_RATE_HZ, 0.01f * gyroConfig()->dyn_notch_quality, FILTER_BPF);
```

Comment:
Are you sure that this does not require the default to be scaled as well? With this change, we are changing from a hardcoded 0.05 that it was pre #6421 to 0.7 as a default, which is somewhat suspicious.

Reply:
The original default was 1/sqrt2, or 0.707

Reply:
I tested 0.2, then 0.1, then 0.05... and looked at them using the simulator here, which showed 0.05 was a good setting to make the biquad work as a proper bandpass:
http://www.earlevel.com/main/2013/10/13/biquad-calculator-v2/

Reply:
All good then.